segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/StatsReporter.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.chunklayer;
+
+/**
+ * Contract for components that reports stats.
+ */
+public interface StatsReporter {
+    default void report() {
```

Comment:
Javadoc .

Also, why do you give it a noop default implementation if you have a single method? If a class does not have anything to report, then maybe it shouldn't implement this interface in the first place.

Reply:
fixed.